db/seeds.rb

```diff
@@ -20,4 +20,83 @@
                email: email,
                password:              password,
                password_confirmation: password)
-end
\ No newline at end of file
+end
+
+
+# Make a rhizome
+Rhizome.create!(
+  name: "Test rhizome",
+  particle_device_attributes: {
+    device_id: Faker::Lorem.characters(24),
+    encrypted_access_token: "S3jF/ramSAL8dZ4zzASXeuBunWv6tLn9jzcU4ABEflfGQkrwbM44CpWMtyRhkED/"
+    }
+  )
+
+
+# Make an equipment profile
+EP = EquipmentProfile.create!(
+  name: "Test EP")
+
+eq1 = HeatingElement.create!(
+  power_pin: 1,
+  control_pin: 0,
+  equipment_profile: EP
+  )
+
+eq2 = TemperatureSensor.create!(
+  data_pin: 2,
+  onewire_index: 10,
+  equipment_profile: EP
+  )
+
+eq3 = Thermostat.create!(
+  equipment_profile: EP,
+  element: HeatingElement.create!(
+    power_pin: 3,
+    control_pin: 5,
+    equipment_profile: EP
+    ),
+  sensor: TemperatureSensor.create!(
+    data_pin: 4,
+    onewire_index: 9,
+    equipment_profile: EP
+    )
+  )
+  
+
+# Make a schedule and tasks
+sched = Schedule.create!(
+  name: "Test Sched 1"
+  )
+
+t1 = Task.create!(
+  type: "TurnOnTask",
+  status: :queued,
+  update_data: {"state": "ON"},
```

Comment:
The :update_data for Task subclasses is similar to :pins for Equipment subclasses. I didn't catch this earlier, but you should do a similar thing here and set :state rather than passing it in as a subhash.